_includes/newsletter-modal.html

```diff
@@ -4,8 +4,8 @@
             <div class="modal-body">
                 <img src="/assets/img/guides/layout/img-newsletter.png" alt="newsletter_image" /><br />
                 <label class="modal-title" id="newsletterModalLabel">Success! You’re signed up. </label>
-                <p>You signed up for our newsletter with the email address: </p>
-                <button type="button" class="btn btn-secondary" data-dismiss="modal">Got it</button>
+                <p>You signed up for our newsletter </p>
+                <button type="button" class="btn btn-secondary" onclick="window.history.pushState({path: window.location.pathname },'',window.location.pathname)" data-dismiss="modal">Got it</button>
```

Comment:
Why do you need this `onclick` behavior here?

Reply:
I added it to clear the URL to point to the current path once the button is clicked
E.g When we have the additional parameters in the URL `newsletter_success=1`, after clicking on the `Got it` button which dismisses the modal, the URL remains the same having the url like `/guides?newsletter_success=1. So I did that to ensure that only the pathname is displayed without the parameters.

Reply:
I see. Can you add that explanation as a comment?